client/src/routes/fight/modal/FightSummary.svelte

```diff
@@ -1,12 +1,8 @@
 <script lang="ts">
-  import { getContext } from 'svelte';
-  import { FightSocket, key } from '../../FightSocket';
-
-  const socket = (getContext(key) as () => FightSocket)();
+    import {replace} from "svelte-spa-router";
 
     function leave(){
-        socket.close();
-        window.location.reload();
+        replace("/fight")
```

Comment:
Pytanie, czy nie chcielibyśmy wrócić do ekranu głównego

Reply:
Założyłem, że jeśli ktoś sędziował w walce (i dostał się do ekranu podsumowania), to znaczy, że jest sędzią i po walce jeśli będzie chciał dalej korzystać z aplikacji, to podłączy się do innej walki, a nie przejdzie na ekran organizatora. Z drugiej strony nie wiem czy jest sens ograniczać tak użytkownika, może zdarzy się sędzia-organizator.

Reply:
Można pomyśleć w takim razie o dwóch ścieżkach: następna walka albo ekran główny